src/open_inwoner/scss/components/Typography/P.scss

```diff
@@ -1,63 +1,64 @@
-.p {
-  color: var(--font-color-body);
-  font-family: var(--font-family-body);
-  font-size: var(--font-size-body);
-  line-height: var(--font-line-height-body);
-  font-weight: normal;
-  margin: 0;
+@import '~@utrecht/components/dist/paragraph/css/index.css';
 
-  &#{&}--compact {
-    margin-top: 0 !important;
-    margin-bottom: 0 !important;
-  }
+.utrecht-paragraph {
+  // Takes style values from NLDS community
 
-  &--small {
-    font-size: var(--font-size-body-small);
-    line-height: var(--font-line-height-body-small);
-  }
+  &--oip {
+    // Takes style values from our own design-tokens NPM package
+    margin: 0;
 
-  &--muted {
-    color: var(--color-mute);
-  }
+    &-small {
+      font-size: var(--oip-utrecht-paragraph-small-font-size);
```

Comment:
I would prefer variable names where the size indicator is at the end, e.g. `oip-utrecht-paragraph-font-size-small`, `oip-utrecht-paragraph-font-size-medium` etc., as this would be easier to discern.

Reply:
This is not about 'preference', this is how NLDS SCSS variables are built. In this case I am making a variant of the already existing "**paragraph-small**" so that is actually its entire name. 
Indeed note that the SCSS variables/values do not quite follow the same order as HTML classes.
So they look like they a different hierarchy than the usual BEM HTML classes, but they actually don't. It looks confusing because the HTML classnames look a lot like their CSS variables, but not quite...

Here in these screenshots you can see examples of Open-Formulieren and Amsterdam and how it will make themeswitching easier in the future if we keep the NLDS order intact. I know this looks confusing... And Open-Forms actually creates local variables for their own custom components, which is something we could also do, but I decided to not do that and just create these as coming from our own design-tokens.

<img width="1503" alt="NLDS-amsterdam-order" src="https://github.com/maykinmedia/open-inwoner/assets/118177951/65c7d6ab-38d8-4e8c-bf88-c1b3a3eedf28">

![NLDS-OF-order](https://github.com/maykinmedia/open-inwoner/assets/118177951/0fa0c0bd-bbce-4d16-b483-36c8a2569dea)

Reply:
Whoever decided to build the variables this way made the wrong choice IMO, but oh well....